code/modules/mob/living/simple_animal/hostile/ordeal/steel.dm

```diff
@@ -230,20 +220,24 @@
 	deathmessage = "mutters something under their breath before collapsing."
 	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID|MOB_BUG
 	ranged_cooldown_time = 15 SECONDS
+	move_to_delay = 3
 	maxHealth = 2000
 	health = 2000
 	melee_damage_lower = 40
 	melee_damage_upper = 57
 	vision_range = 12
 	ranged = TRUE
-	retreat_distance = 3
 	minimum_distance = 4
-	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 1)
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.4, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.6, PALE_DAMAGE = 0.6)
 	can_patrol = TRUE
 	wander = 0
 	footstep_type = FOOTSTEP_MOB_SHOE
 	possible_a_intents = list(INTENT_HELP, INTENT_GRAB, INTENT_DISARM, INTENT_HARM)
 	butcher_results = list(/obj/item/food/meat/slab/human = 2, /obj/item/food/meat/slab/human/mutant/moth = 1)
+	/// Amount of WHITE damage done by screech attack
+	var/screech_damage = 120
+	/// Do_after time delay for screech
+	var/screech_delay = 5 SECONDS
```

Comment:
Why have a variable if it isnt going to be edited later on?

Reply:
Because it lets us edit it later if we're feeling like it, without searching for where it is used.
Additionally, lets admins to change it in-game if they want to.